isovar/locus_reads.py

```diff
@@ -29,17 +28,34 @@
 )
 from .common import list_to_string
 from .dataframe_builder import DataFrameBuilder
-
+from .value_object import ValueObject
 
 logger = logging.getLogger(__name__)
 
-class LocusRead(namedtuple("LocusRead", [
+class LocusRead(ValueObject):
+    __slots__ = [
         "name",
         "sequence",
         "reference_positions",
         "quality_scores",
         "base0_read_position_before_variant",
-        "base0_read_position_after_variant"])):
+        "base0_read_position_after_variant"
+    ]
+
+    def __init__(
```

Comment:
@iskandr: I am all for following the guideline, but it would be really hard to enforce it if we can't use lint-type-of-tools to verify the new code against those rules and make these checks part of the CI tests. This is just a practicality issue, though. :+1: